admin/app/templates/authenticated/certifications/index.hbs

```diff
@@ -8,7 +8,7 @@
   <section class="page-section">
     <ul class="certifications-menu">
       <li>
-        {{#link-to "authenticated.certifications.sessions"}}
+        {{#link-to "authenticated.sessions"}}
```

Comment:
Allez, on s'octanise !
https://guides.emberjs.com/release/routing/linking-between-routes/

Reply:
Je pense que ça sera fait dans la seconde PR vu que ça concernera cette route , je prends note

Reply:
Ouais mais bon, franchement c'est pas grand chose. Ca mérite pas une PR. Surtout que là tu l'appliques juste sur ces deux nouvelles routes, pas sur le reste. Le reste en effet on verra plus tard, mais sur toutes les nouveautés que tu introduis (même recopiés d'ailleurs), faisons l'effort d'octaniser

Reply:
Bah en fait ce fichier là va totalement disparaître on aura plus ces links-to... après je peux quand même les octaniser xD